src/index.js

```diff
@@ -0,0 +1,153 @@
+			},
+		})
+		.then((response) => {
+			// console.log(response.data.main.temp);
+			state.temp = Math.round(convertKtoF(response.data.main.temp));
+			updateTemp();
+			updateTempColor();
+			updateLandscape();
+		})
+		.catch((error) => {
+			console.log('error!', error.response.data);
+		});
+};
+
+// Sets the temperate value to state
+const updateCityName = () => {
+	const headerCityName = document.querySelector('#headerCityName');
+	state.city = cityNameInput.value;
+	headerCityName.textContent = state.city;
+};
+
+const updateTemp = () => {
+	tempValue.textContent = `${state.temp}`;
+};
+
+const updateTempColor = () => {
+	tempValue.classList.remove('red', 'orange', 'yellow', 'green', 'blue');
+	if (state.temp >= 80) {
+		tempValue.classList.add('red');
+	} else if (state.temp >= 70 && state.temp <= 79) {
+		tempValue.classList.add('orange');
+	} else if (state.temp >= 60 && state.temp <= 69) {
+		tempValue.classList.add('yellow');
+	} else if (state.temp >= 50 && state.temp <= 59) {
+		tempValue.classList.add('green');
+	} else if (state.temp < 50) {
+		tempValue.classList.add('blue');
+	}
+};
+
+const updateLandscape = () => {
+	if (state.temp >= 80) {
+		landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+	} else if (state.temp >= 70 && state.temp <= 79) {
+		landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+	} else if (state.temp >= 60 && state.temp <= 69) {
+		landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+	} else if (state.temp < 59) {
+		landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+	}
+};
+
+const updateSky = () => {
+	const skySelect = document.querySelector('#skySelect').value;
+	const skyContainer = document.querySelector('#sky');
+	let sky = '';
+	let skyColor = '';
+
+	if (skySelect === 'Sunny') {
+		sky = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+		skyColor = 'sunny';
+	} else if (skySelect === 'Cloudy') {
+		sky = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+		skyColor = 'cloudy';
+	} else if (skySelect === 'Rainy') {
+		sky = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+		skyColor = 'rainy';
+	} else if (skySelect === 'Snowy') {
+		sky = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+		skyColor = 'snowy';
+	}
+	skyContainer.textContent = sky;
+	const gardenContent = document.querySelector('#gardenContent');
+	gardenContent.classList = `garden__content ${skyColor}`;
+};
+
+const increaseTemp = () => {
+	state.temp += 1;
+	updateTemp();
+	updateTempColor();
+	updateLandscape();
+};
+
+const decreaseTemp = () => {
+	state.temp -= 1;
+	updateTemp();
+	updateTempColor();
+	updateLandscape();
+};
+
+const resetCityName = () => {
+	const cityNameResetButton = document.querySelector('#cityNameReset');
+	cityNameInput.value = 'South Lake Tahoe';
+	updateCityName();
+};
+
+updateCityName();
+// Registers functions as 'event listeners'
+increaseTempButton.addEventListener('click', increaseTemp);
+decreaseTempButton.addEventListener('click', decreaseTemp);
+cityNameInput.addEventListener('input', updateCityName);
+currentTempButton.addEventListener('click', getLatAndLong);
+skySelect.addEventListener('change', updateSky);
```

Comment:
Same as above! Consider adding these event handlers to a function that loads everything once the DOM is completely rendered